### PR TITLE
improve (marzneshin-dashboard-public-locales-fa.json): farsi translation

### PR DESCRIPTION
**Pull request by realartin**
…nslation

adding subscription titles and other buttons farsi traslation

## Description

Provide a small description of Pull Request.

## UI Changes

-
-

## Checklist
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests (stories, interaction tests, unit tests, e2e tests) to cover my changes.
- [ ] I have added tests to cover my changes.

## Related Issues

Closes

- #
- #

### CODE REVIEW

**Review by realartin — COMMENTED** [review]
improve (marzneshin-dashboard-public-locales-fa.json): farsi translation

File: dashboard/public/locales/fa.json [hunk 1]
```diff
@@ -89,9 +89,9 @@
         "update-interval": "بازه آپدیت",
         "add-rule": "افزودن قانون",
         "reset-local-changes": "بازنشانی",
-        "title": "تنظیمات اشتراک",
+        "title": "اشتراک",
         "subscription-title": "قوانین اشتراک",
-        "subscription-desc": "قوانین اشتراک خود را به ترتیب اولویت تنظیم کنید.",
+        "subscription-desc": "تنظیمات اشتراک را با قوانین قابل تنظیم و گزینه‌های به‌روزرسانی مدیریت و سفارشی‌سازی کنید.",
```
Comment (erfjab) [inline dashboard/public/locales/fa.json:94]: تنظیمات اشتراک را با قوانین قابل تنظیم و گزینه‌های به‌روزرسانی، مدیریت و شخصی‌سازی کنید.

File: dashboard/public/locales/fa.json [hunk 2]
```diff
@@ -173,6 +173,15 @@
...
       "usage_duration": "مدت استفاده",
       "activation_deadline": "انقضای تعلیق",
+      "nodes-usage": "مصرف از نودها",
+      "settings": {
+        "nodes-usage": {
+          "last-days": "روز  {{count}} آخرین",
+          "last-months": "ماه  {{count}} آخرین",
+          "last-hours": "ساعت  {{count}} آخرین",
+          "title": "مصرف کاربر از نود ها",
+          "desc": "میزان مصرف کاربر از نود ها در زمان مشخص ."
```
Comment (erfjab) [inline dashboard/public/locales/fa.json:183]: میزان مصرف کاربر از هر یک از نودها در طول زمان مشخص.

**Review by realartin — COMMENTED** [review]
improve (marzneshin-dashboard-public-locales-fa.json): farsi translation

**Review by realartin — COMMENTED** [review]
the last fix

**Review by erfjab — APPROVED** [review]

**Review by realartin — COMMENTED** [review]
Done

**Review by khodedawsh — APPROVED** [review]

**Pull request merged**